homeassistant/components/min_max/sensor.py

```diff
@@ -98,19 +107,20 @@ async def async_setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up the min/max/mean sensor."""
-    entity_ids = config.get(CONF_ENTITY_IDS)
-    name = config.get(CONF_NAME)
-    sensor_type = config.get(CONF_TYPE)
-    round_digits = config.get(CONF_ROUND_DIGITS)
+    entity_ids: list[str] = config[CONF_ENTITY_IDS]
+    name: str | None = config.get(CONF_NAME)
+    sensor_type: str = config[CONF_TYPE]
+    round_digits: int = config[CONF_ROUND_DIGITS]
+    unique_id: str | None = config.get(CONF_UNIQUE_ID)
 
     await async_setup_reload_service(hass, DOMAIN, PLATFORMS)
 
     async_add_entities(
-        [MinMaxSensor(entity_ids, name, sensor_type, round_digits, None)]
+        [MinMaxSensor(entity_ids, name, sensor_type, round_digits, unique_id)]
     )
 
 
-def calc_min(sensor_values):
+def calc_min(sensor_values: list[tuple[str, Any]]):
```

Comment:
All methods that added typing (this one and the ones below), are all not fully typed, they miss a typed return value.

Reply:
Fixed